src/components/button/_button.scss

```diff
@@ -70,6 +70,7 @@
     // text when focussed.
     @include govuk-compatibility(govuk_template) {
       &:link:focus {
+        outline-color: transparent;
```

Comment:
In a different part of the codebase I found I could just remove this compatibility stuff since it isnt needed anymore, might be the same here, worth checking.